python/lsst/sims/catUtils/mixins/PhotometryMixin.py

```diff
@@ -148,6 +148,22 @@ def get_lsst_photometric_uncertainties(self):
                                                  'lsstBandpassDict')
 
 
+    def calculateVisibility(self, magFilter, m5, sigma=0.12):
+        """
+        Calculate the probability of detecting a particular source.
```

Comment:
I'm not sure that the probability column would ever be useful. I will update the doc string to reflect what it does return more accurately.